features/steps/steps.rb

```diff
@@ -172,3 +170,39 @@ def get_version_number(file_path)
   Maze.check.not_include(run_output, "[INFO]")
   Maze.check.not_include(run_output, "[DEBUG]")
 end
+
+Before('@installation') do
+  @base_dir = Dir.pwd
+  @output = `npm pack`
+  @bugsnag_cli_package_path = "#{@base_dir}/#{@output}"
+end
+
+When('I install the bugsnag-cli via {string} in a new directory') do |package_manager|
+  case package_manager
+  when 'npm'
+    @fixture_dir = "#{@base_dir}/features/cli/fixtures/#{package_manager}"
+    Dir.mkdir(@fixture_dir)
+    Dir.chdir(@fixture_dir)
```

Comment:
This can be moved outside the `case` block I think?

(And the final `chdir`)

Reply:
I've refactored this block of code to try and make it as DRY as possible.